src/OrchardCore.Modules/OrchardCore.OpenId/Views/OpenIdServerSettings.Edit.cshtml

```diff
@@ -126,7 +126,23 @@
     <span class="hint">@T["The base URL of the identity server (this site). If none is provided, a default value based on the site host is automatically computed."]</span>
 </div>
 
-<div class="mb-3" asp-validation-class-for="EncryptionCertificateStoreLocation">
+<div class="mb-3" asp-validation-class-for="EncryptionRsaSecret">
+    <label asp-for="EncryptionRsaSecret" class="form-label">@T["Encryption RSA secret"]</label>
+    @await Component.InvokeAsync(
```

Comment:
nb: this component is not available unless the feature is on.

Currently it's written, like the background tasks feature, where the secrets services always exist, and are always on, and the secrets module, provides ui only, and is not required on.

However, for the component to be found, we need the module on, or have open id take a dependency on it (probably latter)

Something to tweak